test/Audio/SoundStream.test.cpp

```diff
@@ -8,7 +8,7 @@
 
 namespace
 {
-class SoundStream : public sf::SoundStream
+struct SoundStream : sf::SoundStream
```

Comment:
I'd stay consistent and not do this. Also most of these methods should actually remain private in their implementation.